docs/releases/1.10.txt

```diff
@@ -202,6 +202,9 @@ Database backends
 ~~~~~~~~~~~~~~~~~
 
 * Temporal data subtraction was unified on all backends.
+* Added ``**kwargs`` to ``as_sql`` method of Expressions to override behavior
+  through arguments rather than mutating ``self`` itself, which is not safe when using
+  multiple databases with different backends.
```

Comment:
... " rather than mutating ``self``, which is not safe when using multiple databases with different backends."